CHANGELOG.md

```diff
@@ -7,9 +7,14 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 [Unreleased][unreleased]
 -------------------------------------------------------------------------
 ### Changed
+
 - Increase required version of the cryptography package to >=1.4.0.
+
 ### Fixed
+
 - Remove uses of deprecated functions from the cryptography package.
+- Warn about missing `algorithms` param to `decode()` only when `verify` param is `True` [#281][281]
```

Comment:
This is missing defining the link for 281 at the bottom of the file.

Reply:
Thanks @jpadilla! That's fixed now.